### ISSUE TITLE
Build fail

### ISSUE THREAD

**Issue by stingus** [eventuate-examples/eventuate-examples-java-customers-and-orders#14]
When I run build-and-test-all.sh, the script fails with:
```
Could not resolve all dependencies for configuration ':customers-service:testCompileClasspath'.
> Could not find net.chrisrichardson.eventstore.examples.customersandorders:common-contracts:1.0-SNAPSHOT.
  Required by:
      project :customers-service
```

**Comment by cer** [comment]
Investigating...

**Comment by cer** [comment]
Fixed by 5d19154c2927776a96c04517f43b0f17a9cbb48d
